.gitmodules

```diff
@@ -1,15 +1,12 @@
 [submodule "FV3"]
 	path = FV3
 	url = https://github.com/NOAA-EMC/fv3atm
-	branch = develop
```

Comment:
please do not remove this "develop" branch. Some repo the default branch is "master", removing this line may cause problem.

Reply:
FV3 branch was specified as "orion", which does not exist any more.
I will add "develop" to FV3, NEMS, CIC_SRC, MOM6, and WW3.